MetadataExtractor/KeyValuePair.cs

```diff
@@ -6,17 +6,11 @@ namespace MetadataExtractor
     /// Models a key/value pair, where both are non-null <see cref="string"/> objects.
     /// </summary>
     /// <author>Drew Noakes https://drewnoakes.com</author>
-    public sealed class KeyValuePair
+    public sealed class KeyValuePair(string key, StringValue value)
```

Comment:
Wondering if this should be a struct or even a record.

Reply:
Updated to a readonly struct